src/app/onion/learning-object-builder/components/user-dropdown/user-dropdown.component.ts

```diff
@@ -87,13 +87,6 @@ export class UserDropdownComponent implements OnInit, DoCheck, OnDestroy {
   findUser(query: string) {
     if (query && query !== '') {
       this.userService.searchUsers({ text: query }).then((results: User[]) => {
-        // remove current user from results
-        for (let i = 0; i < results.length; i++) {
```

Comment:
You know what's funny is that Justine or anyone of us could go through and add the author in anyway we don't have logic blocking that since this is only barring the person logged in from being added.

Reply:
Yeah but it's easier to script 700+ objects than to create a new revision of each lol.

Reply:
I completely agree. I think it's funny that this was the only bit of code that blocked us from having our new workflow from the beginning.